src/validation.cpp

```diff
@@ -763,7 +763,7 @@ bool MemPoolAccept::PreChecks(ATMPArgs& args, Workspace& ws)
 
     ws.m_iters_conflicting = m_pool.GetIterSet(ws.m_conflicts);
     // Calculate in-mempool ancestors, up to a limit.
-    if (ws.m_conflicts.size() == 1) {
+    if (ws.m_conflicts.size() == 1 && HasNoNewUnconfirmed(tx, m_pool, ws.m_iters_conflicting)) {
```

Comment:
Would it be worth documenting the CPFP carve-out rule (in the `doc` folder, similarly to RBF)?
Especially now that it does require no new unconfirmed inputs which RBF doesn't?
It can be done in a separate PR though.

Reply:
Sorry it took me so long to get to this - added in #23711